### ISSUE TITLE
Why not Pakistan

### ISSUE THREAD

**Issue by ehsansajjad465** [msdxbelux/XamarinAlliance#15]
I am a developer from Pakistan, just curious to know how developers from Pakistan can participate in XamarinAlliance challenges ?

**Comment by lilepeeps** [comment]
Hi there. 

You can participate in Xamarin Alliance from any country in the world, Pakistan included! However the diploma for completing the challenges is no longer available.

Peter

**Comment by ehsansajjad465** [comment]
What is the reason for not availability of Diploma on completion of it.

**Comment by icebeam7** [comment]
Because June 30 was the last day to complete the challenges,

**Comment by ehsansajjad465** [comment]
oh alright... so i am late in the party!!

**Comment by icebeam7** [comment]
Yeah, however I strongly recommend you to go through and solve them. Trust me, you'll learn a lot :) The knowledge you earn and the learning experience are more valuable than anything else.

**Comment by ehsansajjad465** [comment]
no doubt in that and yes i will as i am looking forward to experiment with Xamarin.
